context/app/static/js/pages/Templates/Templates.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import Stack from '@mui/material/Stack';
+
+import { TemplateGridPreview, TemplateLogInPanel } from 'js/components/workspaces/workspaceMessaging';
+import { WorkspacesIcon } from 'js/shared-styles/icons';
+import IconPageTitle from 'js/shared-styles/pages/IconPageTitle';
+
+function Templates() {
+  return (
+    <Stack spacing={3}>
+      <IconPageTitle icon={WorkspacesIcon}>Templates</IconPageTitle>
+      {!isAuthenticated && <TemplateLogInPanel />}
```

Comment:
We could also use the `!isAuthenticated` as a guard in the template login panel itself if it's meant to only be visible to unauthenticated users; this appears to be the pattern used in the `LogInPanel` below as well.